src/rgw/setup/rgw.py

```diff
@@ -827,6 +827,22 @@ def _logrotate_generic(conf: MappedConf):
             Log.info(f'{const.LOGROTATE_TMPL} file copied to {const.LOGROTATE_CONF}')
         except Exception as e:
             Log.error(f"Failed to configure logrotate for {const.COMPONENT_NAME}. ERROR:{e}")
+
+        # Copy log rotate script for rgw core files into system's logrotate directory.
+        old_core_logrotate_file = os.path.join(const.LOGROTATE_DIR, const.CORE_LOGROTATE_FILE_NAME)
```

Comment:
why handling these old files? any fresh deployment wont have old files right.

Reply:
added for additional validation. but yes your point is valid, we can safely remove this.

Reply:
done